extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1026,7 +1141,83 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
 
   getPreconnectUrls() {
     return ['https://partner.googleadservices.com',
-      'https://tpc.googlesyndication.com'];
+      SAFEFRAME_ORIGIN];
+  }
+
+  /** @override */
+  getNonAmpCreativeRenderingMethod(headerValue) {
```

Comment:
Now that I think of it, we don't actually need to enforce this on the client given we can just ensure fluid traffic gets the safeframe header.  It may make the code simpler... let's discuss offline

Reply:
As discussed offline, keeping this for the time being.